domain/ide/command/open_browser.go

```diff
@@ -19,30 +19,23 @@ package command
 import (
 	"context"
 
+	"github.com/rs/zerolog/log"
 	"github.com/snyk/go-application-framework/pkg/auth"
 
 	"github.com/snyk/snyk-ls/domain/snyk"
 )
 
-type OpenBrowserCommand struct {
-	command snyk.Command
+type openBrowserCommand struct {
+	command snyk.CommandData
 }
 
-func NewOpenBrowserCommand(url string) *OpenBrowserCommand {
-	return &OpenBrowserCommand{
-		command: snyk.Command{
-			Title:     snyk.OpenBrowserCommand,
-			CommandId: snyk.OpenBrowserCommand,
-			Arguments: []any{url},
-		},
-	}
-}
-
-func (cmd *OpenBrowserCommand) Command() snyk.Command {
+func (cmd *openBrowserCommand) Command() snyk.CommandData {
 	return cmd.command
 }
 
-func (cmd *OpenBrowserCommand) Execute(_ context.Context) error {
-	auth.OpenBrowser(cmd.command.Arguments[0].(string))
+func (cmd *openBrowserCommand) Execute(_ context.Context) error {
+	url := cmd.command.Arguments[0].(string)
+	log.Debug().Str("method", "openBrowserCommand.Execute").Msgf("opening browser url %s", url)
+	auth.OpenBrowser(url)
```

Comment:
Why auth encapsulates open browser command? How this command is related to auth at all? 👀

Reply:
go application framework has the command in its auth package. We just use it to remove duplication.